### PR TITLE
Create unesp-dracena.csl

### PR DESCRIPTION
**Pull request by bdrunesp**
Style was configured by the Library of the Faculty of Agrarian and Technological Sciences (UNESP - FCAT - Faculdade de Ciências Agrárias e Tecnológicas) in the author-date system of the technical standard of the Brazilian Association of Technical Standards (ABNT) entitled NBR 10520: 2002 - Information and documentation - Citations in documents - Presentation and the references are configured according to NBR 6023: 2002 - Information and documentation - References - Elaboration

### CODE REVIEW

File: unesp-dracena.csl [hunk 1]
```diff
@@ -0,0 +1,589 @@
...
+      <term name="month-03" form="short">mar.</term>
+      <term name="month-04" form="short">abr.</term>
+      <term name="month-05" form="short">maio</term>
+      <term name="month-06" form="short">jun.</term>
+      <term name="month-07" form="short">jul.</term>
+      <term name="month-08" form="short">ago.</term>
+      <term name="month-09" form="short">set.</term>
+      <term name="month-10" form="short">out.</term>
+      <term name="month-11" form="short">nov.</term>
+      <term name="month-12" form="short">dez.</term>
```
Comment (POBrien333) [inline unesp-dracena.csl]: All these month locales are already set like that via the language setting. You'd only set these if you'd wanted to override them. See here for the pt-BR locale file: https://github.com/citation-style-language/locales/blob/master/locales-pt-BR.xml

File: unesp-dracena.csl [hunk 2]
```diff
@@ -0,0 +1,589 @@
...
+            <text macro="author" suffix=". "/>
+            <text macro="title" suffix=". "/>
+            <text macro="edition" suffix=". "/>
+            <text macro="place" suffix=", "/>
+            <text macro="issued" suffix=". "/>
+            <text macro="access" suffix=". "/>
+            <text macro="archive" suffix=". "/>
+          </group>
+        </else-if>
+        <else-if type="webpage">
```
Comment (POBrien333) [inline unesp-dracena.csl]: we usually group webpages and post-weblogs together as they're both web sources.

File: unesp-dracena.csl [hunk 3]
```diff
@@ -0,0 +1,589 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="never" default-locale="pt-BR">
+  <info>
+    <title>UNESP - Campus de Dracena - ABNT - Autores abreviados (Portuguese - Brazil)</title>
+    <title-short>UNESP-FCAT</title-short>
+    <id>http://www.zotero.org/styles/unesp-dracena</id>
+    <link href="http://www.zotero.org/styles/unesp-dracena" rel="self"/>
+    <link href="http://www.zotero.org/styles/unesp-dracena.csl" rel="template"/>
```
Comment (POBrien333) [inline unesp-dracena.csl]: the test is failing as this is not a template style. Either include the style you used as a base for this here style or remove line 8. (better to include the link to the template style as this gives us a history and also honours the work of the maker of the template style.)
Also, no need to submit this 4 times, you can just edit the one style here and close the other submissions.

**Comment by rmzelle** [comment]
Replaces #3541, #3542, #3543, and #3545.

**Comment by rmzelle** [comment]
@bdrunesp, I made some changes to the style title and file name to make them more consistent with our other styles.

We would like to add a "template" link that points to the style you started out from (see also https://github.com/citation-style-language/styles/wiki/Style-Requirements#8---template-link). Can you tell us which style that is?

**Comment by rmzelle** [comment]
And the post above has instructions on how to update the current pull request (see https://github.com/citation-style-language/styles/pull/3544#issuecomment-395849290), so there is no need to create additional pull requests. We can just update this one until it is ready to be accepted.

**Comment by bdrunesp** [comment]
@rmzelle  Thank you very much for your help.

**Comment by bdrunesp** [comment]
@rmzelle Our style was based on the template: http://www.zotero.org/styles/associacao-brasileira-de-normas-tecnicas-ufrgs

**Comment by rmzelle** [comment]
Thanks!

**Pull request merged**